lib/blpapi-wrapper.ts

```diff
@@ -69,15 +69,30 @@ var EVENT_TYPE = {
   , PARTIAL_RESPONSE: 'PARTIAL_RESPONSE'
 };
 
+type StringToStringMap = {
+    [s: string]: string;
+};
+
+var REQUEST_TO_RESPONSE_MAP: StringToStringMap = {
```

Comment:
I would vote for using a direct type definition instead of type alias for simplicity:
```typescript
var REQUEST_TO_RESPONSE_MAP: {[index: string]: string}
```